src/EnergyPlus/SQLiteProcedures.cc

```diff
@@ -1221,7 +1221,6 @@ void SQLite::initializeTabularDataTable()
                              "SimulationIndex INTEGER, "
                              "RowId INTEGER, "
                              "ColumnId INTEGER, "
-                             "Value TEXT "
```

Comment:
So this line was added as part of a large performance refactor from @mbadams5 (#4610).  This line does look wrong, though I'll say from inspection it looks more like it should just have a comma added rather than the line being removed entirely.  I'm not saying this patch is wrong, just saying I may have guessed wrong.  I'll need to pull the database and look at it to verify.  

@mbadams5 or @jmarrec if either of you have a minute to also peek at this, it would be appreciated.